src/main/java/com/commercetools/sync/types/utils/TypeUpdatePlainEnumActionUtils.java

```diff
@@ -0,0 +1,84 @@
+package com.commercetools.sync.types.utils;
+
+import io.sphere.sdk.commands.UpdateAction;
+import io.sphere.sdk.models.EnumValue;
+import io.sphere.sdk.types.Type;
+import io.sphere.sdk.types.commands.updateactions.AddEnumValue;
+import io.sphere.sdk.types.commands.updateactions.ChangeEnumValueOrder;
+
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import static com.commercetools.sync.types.utils.TypeUpdateEnumActionsUtils.buildAddEnumValuesUpdateActions;
+import static com.commercetools.sync.types.utils.TypeUpdateEnumActionsUtils.buildChangeEnumValuesOrderUpdateAction;
+import static java.util.Collections.emptyList;
+
+public final class TypeUpdatePlainEnumActionUtils {
```

Comment:
I would also name it: `PlainEnumUpdateActionUtils`

Reply:
https://github.com/commercetools/commercetools-sync-java/commit/6f1484541de42487b69083d4556201bc2cb40080#diff-32d75125deae33b8dc07b973a10de669L20